jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,107 @@
+
+`curl -u ${CIRCLECI_TOKEN}: https://circleci.com/api/v2/me`
+
+For more detailed information on how to authenticate with a token, please refer to the [Authentication](https://circleci.com/docs/api/#authentication) section in the main CirlceCI API documentation.
+
+### Pipelines
+
+#### What is a Pipeline?
+
+"Pipelines" are a CircleCI designation for the full set of configurations you run when triggering work on your projects on the CircleCI platform. Pipelines contain your workflows, which in turn coordinate your jobs. 
+
+#### What Are the Practical Implications of Pipelines?
+
+In this new paradigm, all jobs will run in a workflow, even if you do not specify one in your configuration. In the future, CircleCI will add a new pipeline-level parameters clause in `config.yml`, referenced in the `pipeline` scope, as well as launch a new version of the CircleCI API that has the ability to trigger pipelines with parameters and retrieve them and their workflows. In addition to these changes, CircleCI is also creating a new User Interface that will unify the experience of viewing your pipelines and their jobs coordinated in workflows.
+
+#### What Does This Change for Me? How Do I Turn On Pipelines?
+
+Whatever you are running today should continue working without any intervention. CircleCI has some features (such as 2.1 configuration and orbs) that require you to enable pipelines. In some rare cases, some users may need to make tweaks to their configuration when enabling pipelines.
+
+Many customers are already using an early version of pipelines, even before CircleCI exposed them through the CircleCI API and web UI. If you have already used the latest features in CircleCI version 2.1 configuration and/or orbs, then you already have pipelines enabled for your project (all projects added after September 1, 2018 already have pipelines enabled by default). If, however, your project does not yet have pipelines enabled, go to "Advanced Settings" in your project's settings page. If you do not see the "Enable Pipelines" option on your project's settings page, this means your project already has pipelines enabled.
+
+#### Is a Pipeline the Same Thing as a Build?
+
+In previous CircleCI release, the term "build" was used to describe a pipeline. Unfortunately, this terminology proved problematic because there was an incongruity between how "build" and "job" were defined in the API and UI, For example, what was referred to as a "build" in past releases is now referred to as a "job" in the current release. To address this ambiguity, CircleCI updated the UI to reflect this change in nomenclature, but there are still instances of "build" in our API.
```

Comment:
```suggestion
In previous CircleCI release, the term "build" was used to describe a pipeline. Unfortunately, this terminology proved problematic because there was an incongruity between how "build" and "job" were defined in the API and UI, for example, what was referred to as a "build" in past releases is now referred to as a "job" in the current release. To address this ambiguity, CircleCI updated the UI to reflect this change in nomenclature, but there are still instances of "build" in our API.
```
Small typo